package.json

```diff
@@ -42,6 +43,8 @@
     "build:full": "npm run clean:lerna && npm install && npm run build",
     "pretest": "npm run clean && npm run build:current",
     "test": "node packages/build/bin/run-nyc npm run mocha --scripts-prepend-node-path",
+    "pretest:ci": "npm run pretest",
```

Comment:
When running on a CI server, I believe we always start with a clean checkout, therefore there is no need to run `npm run clean`?

I am proposing:

```
"pretest:ci": "npm run build",
```

Not a big deal, feel free to land this pull request as it is now.

Reply:
Switched to `npm run build:current`